packages/cli/src/commands/init/editTemplate.ts

```diff
@@ -34,14 +33,6 @@ function replaceNameInUTF8File(
   if (fileContent !== replacedFileContent) {
     fs.writeFileSync(filePath, replacedFileContent, 'utf8');
   }
-
-  if (isPackageJson) {
-    fs.writeFileSync(
-      filePath,
-      fileContent.replace(templateName, projectName.toLowerCase()),
```

Comment:
I think better would be to replace `fileContent` with `replacedFileContent`. That way, we maintain `lowerCase` behaviour, at the same time, we do not overwrite changes (that happened, if any) when working with `package.json` .